creusot/tests/should_succeed/mc91.rs

```diff
@@ -3,7 +3,7 @@ extern crate creusot_contracts;
 use creusot_contracts::*;
 
 #[ensures(x <= 100u32 ==> result == 91u32 &&
-    x > 100u32 ==> result == x - 10u32)]
+    x > 100u32 ==> result@ == x - 10u32)]
```

Comment:
Why does this typecheck? `x` and 10u32 are machine integers, which should no longer have arithmetic operators.

Reply:
I added an implementation `add(i32, i32) -> Int` (and for other integers too). I was trying to keep some of the existing behaviour, but to be honest I would prefer to remove them. If you agree I will do that (and then we would have to write `result@ == x@ - 10`)